packages/lib-classifier/src/store/utils/ClassificationQueue.spec.js

```diff
@@ -109,14 +109,14 @@ describe('ClassificationQueue', function () {
   })
 
   describe('keeps classifications in localStorage if backend fails', function () {
-    let clock
-
     before(function () {
-      clock = sinon.useFakeTimers({ global })
+      sinon.stub(global, 'setTimeout').callsFake(() => 100)
+      sinon.stub(global, 'clearTimeout')
```

Comment:
Similarly, this should mock `clearTimeout` with a function that takes a timeout and clears out. 

By turning off timeouts, you've turned off retries in the classification queue. That's a problem because the tests check whether or not a retry has been scheduled for a given response.